src/cmds/scripts/win_postinstall.py

```diff
@@ -0,0 +1,632 @@
+    if (os.path.exists(postgresql_file_path)):
+        # change port to 15007
+        replace(postgresql_file_path, "#port = 5432", "port = 15007")
+        # change listen address to *
+        replace(
+            postgresql_file_path,
+            "#listen_addresses = 'localhost'",
+            "listen_addresses = '*'")
+        # change escape sequence parsing
+        replace(
+            postgresql_file_path,
+            "#standard_conforming_strings = on",
+            "standard_conforming_strings = on")
+        # change logging collector
+        replace(
+            postgresql_file_path,
+            "#logging_collector = off",
+            "logging_collector = on")
+        # change logging directory
+        replace(
+            postgresql_file_path,
+            "#log_directory = 'pg_log'",
+            "log_directory = 'pg_log'")
+        # change logging filename
+        replace(
+            postgresql_file_path,
+            "#log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'",
+            "log_filename = 'pbs_dataservice_log.%a'")
+        # change logging truncate
+        replace(
+            postgresql_file_path,
+            "#log_truncate_on_rotation = off",
+            "log_truncate_on_rotation = on")
+        # change logging rotation age
+        replace(
+            postgresql_file_path,
+            "#log_rotation_age = 1d",
+            "log_rotation_age = 1440")
+        # change logging format to add timestamp
+        replace(
+            postgresql_file_path,
+            "#log_line_prefix = ''",
+            "log_line_prefix = '%t'")
+        print "postgresql file Successfully configured"
+    else:
+        print "postgresqlfile is not available to configure"
+        exit(7)
+    # Configuring pg_hba.conf file
+    pg_hba_file_path = os.path.join(
+        pbs_path, "home", "datastore", "pg_hba.conf")
+    if (os.path.exists(pg_hba_file_path)):
+        replace(
+            pg_hba_file_path,
+            "# IPv4 local connections:",
+            "# IPv4 connections:")
+        # Change localaddress values for all users
+        original_str = "host    all             all             " + \
+            "127.0.0.1/32            trust"
+        replacement_str = "host    all         all         " + \
+            "127.0.0.1/32          md5\nhost       all         " + \
+            "all        0.0.0.0/0      md5"
+        replace(pg_hba_file_path, original_str, replacement_str)
+        # Change user specific config
+        original_str = "# Allow replication connections from localhost,\
+        by a user with the"
+        replacement_str = "host     all         all         localhost\
+        md5"
+        replace(pg_hba_file_path, original_str, replacement_str)
+        print "pg_hba file Successfully configured"
+    else:
+        print "pg_hba file is not available to configure"
+        exit(8)
+
+
+def InstallDataService(username, password):
+    pbs_pathquote = "\"" + pbs_path
```

Comment:
What the code meant was meant 
_pbs_pathquote = "\"" + pbs_path_
Nothing exceptional